static/js/portico/desktop-redirect.js

```diff
@@ -1,3 +1,2 @@
 const ClipboardJS = require("clipboard");
 new ClipboardJS("#copy");
-document.querySelector("#copy").focus();
```

Comment:
Why was this removed?

Reply:
![Screenshot from 2020-05-09 04-19-53](https://user-images.githubusercontent.com/32801674/81466436-52c3db00-91ef-11ea-9436-357680e8ba6d.png)
It's giving this default highlighted button
Did it break anything?
During testing, it seems to work fine without that

Reply:
The button should be focused on load so the user can use the Enter key to copy the token without reaching for the mouse. Don't remove this.

Reply:
Agree with @andersk.  If for visual reasons, we want to adjust the button styling for that specific button to not show the box, we can just override that styling detail.

Reply:
The focus outline is as important to accessibility as the focus itself. Having the button focused does little good if one can't visually tell that it's focused.

Reply:
makes sense ...I will revert that change